data.py

```diff
@@ -72,7 +72,9 @@ def load_data(config, tokenizer):
         dataset = load_dataset("json", data_files=files, split="train")
 
     else:
-        dataset = load_dataset(dataset_path, split="train")
+        dataset = load_dataset(dataset_path,
+                               split="train",
+                               revision=config["revision"] if "revision" in config else None)
 
     dataset = dataset.train_test_split(test_size=.05, seed=config["seed"])
 
```

Comment:
The suggested change to `remove_columns` seems reasonable and should improve readability by reducing noise in the output data. The addition of `"id"` to this list appears to be a minor correction that ensures consistency in removing unnecessary columns.

In terms of performance, there are no obvious suggestions for improvement in this diff block.